### ISSUE TITLE
Warning: {{image.get_src}} is deprecated and will be removed in 1.1

### ISSUE THREAD

**Issue by lmartins** [MINDKomm/Timmy#5]
You probably already aware of this, but just to be sure a note about a notice Im getting in place of the images once I upgraded to Timber 1.1:

```
Warning: {{image.get_src}} is deprecated and will be removed in 1.1
```

**Comment by gchtr** [comment]
Yes, this will be fixed in the upcoming version.

**Comment by lmartins** [comment]
Ah cool, closing this then.

**Comment by gchtr** [comment]
This was fixed in 2d60a51 and is included in the [v0.10.0 release](https://github.com/MINDKomm/Timmy/releases/tag/v0.10.0).
